queries.sql

```diff
@@ -39,4 +39,50 @@ WHERE NAME != 'Gabumon';
 
 SELECT *
 FROM ANIMALS
-WHERE WEIGHT_KG BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+WHERE WEIGHT_KG BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = 'unspecified';
```

Comment:
- [ ] This should be:
`UPDATE animals SET species = 'pokemon' WHERE species IS NULL;`
You removed unspecified from the species column and so it is not present on your table.